day3/slice_time_image.py

```diff
@@ -20,11 +20,21 @@
 remember to do ``reload(slice_time_script)`` to get the new version.
 """
 
-# Add any extra imports here
-import sys
+# Compatibility settings
+from __future__ import print_function
+from __future__ import division
+from scipy.interpolate import InterpolatedUnivariateSpline
```

Comment:
This is very minor comment - but the standard habit here is to import Python standard libraries such as sys and os first, then very common libraries such as numpy, then less common libraries such as scipy, and nibabel.   It's a tradition that helps the reader look for a given import quickly.